File: easy/smaller_numbers_than_current.go
package easy

func SmallerNumbersThanCurrent(nums []int) []int {
	result := make([]int, len(nums))
	for i := range nums {
		for j := range nums {
			if nums[i] > nums[j] {
				result[i]++
			}
		}
	}
	return result
}

File: easy/smaller_numbers_than_current.py
def smaller_numbers_than_current(nums: list[int]) -> list[int]:
    result = [0] * len(nums)
    for i in range(len(nums)):
        for j in range(len(nums)):
            if nums[i] > nums[j]:
                result[i] += 1
    return result
